packages/client/tsconfig.json

```
@@ -1,8 +1,15 @@
 {
   "compilerOptions": {
+    "noImplicitAny": true,
```

Как смотришь на такие дополнительные ограничения?

```suggestion
    "noImplicitAny": true,
    "alwaysStrict": true, // involves adding the line "use strict" to every compiled file.
    "strictPropertyInitialization": true,
    "strictFunctionTypes": true,
    "noImplicitThis": true,
    "strictBindCallApply": true,
```